modules/cf-core/src/protocol/uninstall-virtual-app.ts

```diff
@@ -549,24 +548,22 @@ async function getUpdatedStateChannelAndAppInstanceObjectsForInitiating(
     stateChannelWithIntermediary,
     stateChannelWithResponding
   ] = [
-    getStateChannelFromMapWithOwners(
+    await getStateChannelFromMapWithOwners(
```

Comment:
same case as install virtual here, because the `getStateChannelFromMapWithOwners` function provides the network as a param, if the `ProxyFactory` bytecode changes in the supplied network context the state channel it pulls from the map could be `undefined` (when we very much expect it to be defined).

the static function should be used in cases where the channel is expected to be properly stored (ie any channel with the intermediary)